tests/middleware/test_cors.py

```diff
@@ -258,7 +258,7 @@ def test_cors_allow_all_methods(test_client_factory):
     )
 
     @app.route(
-        "/", methods=("delete", "get", "head", "options", "patch", "post", "put")
+        "/", methods=["delete", "get", "head", "options", "patch", "post", "put"]
```

Comment:
The fact that we had to edit this makes me think: should we relax the annotation to `typing.Sequence` and then cast it to a list if needed?

Reply:
Thanks for reviewing this.
Good point. I agree.
Let's wait for more feedback then.

Reply:
Oh let me take that back. It will be out of scope for this PR. Let's raise it another one.

Reply:
Yep agreed that it's outside of the scope of this PR.